src/Hazelcast.Net/Exceptions/ClientOfflineException.cs

```diff
@@ -57,7 +57,7 @@ public ClientOfflineException(ClientState state)
         /// </summary>
         /// <param name="message">The message that describes the error.</param>
         [ExcludeFromCodeCoverage]
-        private ClientOfflineException(string message)
+        public ClientOfflineException(string message)
```

Comment:
Why making all public now?

Reply:
because we want to let users catch it in some cases

Reply:
But exception was public before, change is about constructors here.

Reply:
ah, I answered to quickly. when there isn't a `public WhateverException(string message)` we get a warning because the "standard" for exception is to have one such constructor. `private` is not good. so I've made it public, does not hurt and gets rid of a warning.